packages/7zip.plugin.asar.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,26 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = "Asar"
+  $pluginsDir = Join-Path ${Env:ProgramFiles} "7-Zip\Formats"
+  New-Item -ItemType Directory -Force -Path $pluginsDir | Out-Null
+
+  $zipUrl = "https://www.tc4shell.com/binary/Asar.zip"
```

Comment:
this URL does not contain a version and will break with updates.

Reply:
No versioned URL is available  🥲

Reply:
what about install asar using `VM-Install-Node-Tool` instead of the 7zip plugin? Without a versioned URL, this package will eventually break with a tool update. You should be able to create the package using automation (as I did in https://github.com/mandiant/VM-Packages/issues/1048) by reviewing https://github.com/mandiant/VM-Packages/issues/1112, add the `send PR` label to it, and test the created PR label. Alternatively, you can create the PR manually as you did in https://github.com/mandiant/VM-Packages/pull/1140, but using the automation would be a good opportunity to get familiar with it :wink: